python/src/trezorlib/ripple.py

```
@@ -62,10 +62,13 @@ def sign_tx(
 def create_sign_tx_msg(transaction: dict) -> messages.RippleSignTx:
     if not all(transaction.get(k) for k in REQUIRED_FIELDS):
         raise ValueError("Some of the required fields missing")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)